### PR TITLE
SKUPPER-625-podman

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: modules/cli/pages/podman.adoc [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
++
+----
+$ skupper service create backend 8080
+----
+
+. Validate the service from the Podman site by checking the backend API health URL:
++
+--
+----
+$ podman run -it --network=skupper --name=myubi ubi8/ubi curl backend:8080/api/health
```
Comment (fgiorgetti) [inline modules/cli/pages/podman.adoc:166]: Maybe add a `--rm` flag to the `podman run` to ensure container will be removed after using it.

File: modules/cli/pages/podman.adoc [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+Each of the backend replicas respond, for example `Hi, Perfect Parrot. I am Kind Hearted Component (backend-7c84887f9f-wxhxp).`
+
+[NOTE]
+====
+In this scenario, running the `skupper service status` command on the Podman site does not provide much detail about the service:
+
+----
+$ skupper service status
+Services exposed through Skupper:
+╰─ backend (tcp port 8080)
```
Comment (fgiorgetti) [inline modules/cli/pages/podman.adoc:192]: I believe this output is the same you'd get on a Kubernetes site,
for a remote service. 

The target information is only displayed when a given service is bound
to target resources.

File: modules/cli/pages/podman.adoc [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+----
+$ podman inspect --format='{{.NetworkSettings.Networks.skupper.IPAddress}}' hello-world-backend
+
+10.89.0.10
+----
+
+. Expose the `backend` from the Podman site.
++
+----
+$ skupper expose host 10.89.0.10 --address backend --port 8080
```
Comment (fgiorgetti) [inline modules/cli/pages/podman.adoc:230]: You can simply expose the host `hello-world-backend` as the container is running in the same
podman network that skupper is using. That'd make it a bit simpler.

File: modules/cli/pages/podman.adoc [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+. Expose the `backend` from the Podman site.
++
+----
+$ skupper expose host 10.89.0.10 --address backend --port 8080 
+----
+
+. From the Kubernetes site, create the `backend` service:
++
+----
+$ skupper expose host 10.89.0.10 --address backend --port 8080
```
Comment (fgiorgetti) [inline modules/cli/pages/podman.adoc:236]: Here it should be: `skupper service create backend 8080` instead.

**Review by fgiorgetti — COMMENTED** [review]
A few comments, but the overall document looks good!

File: modules/cli/pages/podman.adoc [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+
+
+== Working with services using Skupper podman
+
+The general flow for working with services is the same for Kubernetes and Podman sites.
+However, consider the following:
+
+* Services exposed on Kubernetes are not automatically available to Podman sites.
+This is the equivalent to Kubernetes sites created using `skupper init --enable-service-sync false`.
+* On Kubernetes sites, `skupper service status` shows details about original service location.
```
[the remark this answers is not in the document]
Comment (pwright) [reply modules/cli/pages/podman.adoc:136]: removed, will follow up with question by email

File: modules/cli/pages/podman.adoc (hunk 2 above)
Comment (pwright) [reply modules/cli/pages/podman.adoc:190]: agreed

**Review by fgiorgetti — APPROVED** [review]
LGTM

**Pull request merged**